File: src/main/java/com/wafflestudio/draft/service/RepositoryServiceImpl.java
package com.wafflestudio.draft.service;

import com.wafflestudio.draft.repository.UserRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

@Service
public class RepositoryServiceImpl implements RepositoryService {
    @Autowired
    private UserRepository userRepository;
}

File: src/main/kotlin/com/wafflestudio/draft/service/RepositoryServiceImpl.kt
package com.wafflestudio.draft.service

import com.wafflestudio.draft.repository.UserRepository
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service

@Service
class RepositoryServiceImpl : RepositoryService {
    @Autowired
    private val userRepository: UserRepository? = null
}
